scripts/prepare-release.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+PREVIOUS_VERSION=$(cat deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | grep integreatly-operator | awk -F v '{print $2}')
+
+create_new_csv() {
+  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name integreatly-operator --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+}
+
+set_version() {
+  sed -i "s/$PREVIOUS_VERSION/$VERSION/g" Makefile
+  sed -i "s/$PREVIOUS_VERSION/$VERSION/g" version/version.go
+}
+
+set_images() {
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$SEMVER/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+  sed -i "s/containerImage:.*/containerImage: quay\.io\/integreatly\/integreatly-operator:v$SEMVER/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
```

Comment:
I think we shoudl use TAG here and make sure TAG is set to exactly what we expect a tag to be v1.1.1-er1, v1.1.1 etc..